flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -322,11 +329,22 @@ public Builder cleanupInRocksdbCompactFilter() {
 		 */
 		@Nonnull
 		public Builder cleanupInRocksdbCompactFilter(long queryTimeAfterNumEntries) {
-			cleanupStrategies.activate(CleanupStrategies.Strategies.ROCKSDB_COMPACTION_FILTER,
+			strategies.put(CleanupStrategies.Strategies.ROCKSDB_COMPACTION_FILTER,
 				new RocksdbCompactFilterCleanupStrategy(queryTimeAfterNumEntries));
 			return this;
 		}
 
+		/**
+		 * Enable cleanup of expired state in background.
+		 *
+		 * <p>Depending on actually used backend, the corresponding cleanup will kick in if supported.
+		 */
+		@Nonnull
+		public Builder cleanupInBackground() {
```

Comment:
I suggest we add a simple unit test for this method where `StateTtlConfig` is built like in the doc example with `cleanupInBackground` and test checks that `StateTtlConfig.getCleanupStrategies().isCleanupInBackground()` is `true`, `getIncrementalCleanupStrategy()` is not null, `getIncrementalCleanupStrategy()getCleanupSize()` is 5 etc.